tools/provision-alerts/template/template-adjusted.yml

```diff
@@ -1,7 +1,7 @@
 policies:
   - name: "Canaries metric comparator"
     incident_preference: "PER_POLICY"
-    channels: [1676261]
```

Comment:
Will we get alerts notifications if this is removed?

Reply:
This was done to avoid the whole team getting the notifications. We will not get these but the responsible of the release should check the incidents in NR and detect any anomaly so this shouldn't be a problem.

Reply:
Cool! Thanks for the clarification